metricbeat/module/kafka/_meta/Dockerfile

```diff
@@ -1,18 +1,18 @@
-FROM debian:stretch
+FROM openjdk:8-jre-alpine3.8
 
-ARG KAFKA_VERSION=2.1.1
+ARG KAFKA_VERSION=2.1.0
 
 ENV KAFKA_HOME /kafka
-# The advertised host is kafka. This means it will not work if container is started locally and connected from localhost to it
 ENV KAFKA_LOGS_DIR="/kafka-logs"
 ENV _JAVA_OPTIONS "-Djava.net.preferIPv4Stack=true"
 ENV TERM=linux
 
-RUN apt-get update && apt-get install -y curl openjdk-8-jre-headless netcat dnsutils
+RUN apk add -u bash
```

Comment:
changes to this file and `run.sh` probably belong to a different PR?

Reply:
Changes here could probably be reverted, yes, I think they only reduced the size of the image.

Changes in run.sh are needed, kafka announces the addresses clients should use, current container only works if tests can connect to "kafka:9092". After this change services can be listening on random addresses and ports, so we need some plumbing to let kafka know what address it should announce.
But said that, not all changes are needed now, I got all changes done in #7957, that are intended also to be able to have multiple versions of the same service running at the same time, this is not needed till I merge the Go-part of this PR.

I will reduce the changes in these files to the ones really needed for these changes.